src/javascripts/ng-admin/Crud/repository/CreateQueries.js

```diff
@@ -24,12 +24,14 @@ define(function (require) {
      * @returns {promise}  the new object
      */
     CreateQueries.prototype.createOne = function (view, rawEntity) {
-        return this.Restangular
-            .oneUrl(view.entity.name(), this.config.getRouteFor(view.entity, view.getUrl(), view.type))
-            .customPOST(rawEntity)
-            .then(function (response) {
-                return response.data;
-            });
+
+        var method = view.entity.createMethod(),
+            url = this.Restangular.oneUrl(view.entity.name(), this.config.getRouteFor(view.entity, view.getUrl(), view.type)),
```

Comment:
The variable name isn't well chosen. Use `resource` instead. Same in `UpdateQueries`